ts/call-action-form.tsx

```diff
@@ -104,9 +104,11 @@ export class CallActionForm extends React.Component<Props, State> {
 			} as State);
 			this.props.setModal("loading");
 			var campaign = this.getCampaign(this.props.campaignId);
+			var zipField = (document.getElementsByName('input_zip').item(0) as HTMLInputElement);
```

Comment:
I didn't want to mess with the react class props, so just pull the input value from the document. I believe I did the typescript casting correctly, but please adjust to style.